python/reference-apps/debug-input/debug-input.py

```diff
@@ -0,0 +1,11 @@
+import asyncio
+import json
+from scramjet.streams import Stream
+
+async def delayed_echo(data):
```

Comment:
We lack reference-apps comments also in the Node version, but I think it would be super useful for us later to add even just one comment line to explain why this app was created (we struggle with searching through them in the Node apps)

Reply:
Added a comment.

Reply:
thanks!